futaba/cogs/filter/filter.py

```diff
@@ -38,21 +43,44 @@
 )
 
 
+class SyntheticPattern:
+    __slots__ = ("compiled", "pattern")
+
+    def __init__(self, compiled, pattern):
+        self.compiled = compiled
+        self.pattern = pattern
+
+    def __getattr__(self, item):
+        return getattr(self.compiled, item)
+
+
 class Filter:
     __slots__ = ("text", "regex")
 
     def __init__(self, text):
         logger.info("Creating filter regular expression from %r", text)
-        groups = confusables.is_confusable(text, greedy=True)
-        if groups:
-            pattern = Filter.build_regex(text, groups)
+        if text.startswith("regex:") and len(text) > 6:
+            # Build a general regular expression, de-confusifying LITERALs
```

Comment:
should add a `TODO` for eventually changing this to filter types instead of using prefixes